capstone/backend/src/main/java/com/google/servlets/ShareViaEmailServlet.java

```diff
@@ -0,0 +1,116 @@
+package com.google.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.Properties;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.PasswordAuthentication;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/api/v1/share-via-mailjet")
+public class ShareViaEmailServlet extends HttpServlet {
+
+  private String emailRegEx = "^[^@]+@[^@]+.[^@]+$";
+
+  private class RequestBody {
+    private String email;
+    private String html;
+  }
+
+  private class ResponseBody {
+    private String email;
+
+    public ResponseBody(String email) {
+      this.email = email;
```

Comment:
why do we send the email address back as part of the response?

Reply:
arbitrary response choice, I will change it to a boolean representing success

Reply:
Actually, I am going to have it empty because there is no information in the response that is needed